### DISCUSSION TITLE
What does git pull origin master do?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#237] [Q&A] [answered]
Short answer.

**Comment by montasim** [comment, marked as answer]
The git pull origin master fetches all the changes from the master branch onto the origin and integrates them into the local branch.

git pull = git fetch + git merge origin/ master

After having gone through the beginner level Git interview questions, let us now look at intermediate GIT interview questions and answers.
